app/test/integration_test.go

```diff
@@ -170,10 +178,7 @@ func (s *IntegrationTestSuite) TestMaxBlockSize() {
 				hashes[i] = res.TxHash
 			}
 
-			// wait a few blocks to clear the txs
-			for i := 0; i < 8; i++ {
-				require.NoError(s.network.WaitForNextBlock())
-			}
+			s.WaitForNBlocks(8)
```

Comment:
CI will fail here because the method was renamed

Reply:
resolved by https://github.com/celestiaorg/celestia-app/pull/1430/commits/d429bf8a1aef78bcc0ead859674e8eeb64d7e73f